f5lbaasdriver/v2/bigip/plugin_rpc.py

```diff
@@ -1129,29 +1129,45 @@ def get_devices(self, context, availability_zone):
         with context.session.begin(subtransactions=True):
             try:
                 LOG.info('getting devices begins')
-                devices = self.driver.inventory_plugin.get_devices(
+                devices_db = self.driver.inventory_plugin.get_devices(
                     context,
                     filters={'availability_zone': availability_zone}
                 )
-                LOG.info('getting devices ends: %s' % devices)
+                LOG.info('devices info here: %s' % devices_db)
```

Comment:
Yesterday I saw huge content in neutron server log file, which was produced by monitor. Suggest to shrink the log here. If monitor really require to print device details, it can print it in monitor log file.